custom_components/birthdays/__init__.py

```diff
@@ -115,12 +138,16 @@ async def update_data(self, *_):
 
         days_until_next_birthday = (next_birthday-today).days
 
-        self._age_at_next_birthday = next_birthday.year - self._date_of_birth.year
+        age = next_birthday.year - self._date_of_birth.year
+        self._age_at_next_birthday = age
+        self._extra_state_attributes['age_at_next_birthday'] = age
```

Comment:
Since the attribute key 'age_at_next_birthday' is also present on line 77, extract this to a constant so we don't misspell it (in later refactorings)

Reply:
Not really sure where to do this without breaking it. Any input?